kernel/shell.c

```diff
@@ -231,8 +231,11 @@ static void _mem_command(int argc, char** argv)
     n = __myst_kernel_args.crt_size;
     printf("crt size     =%11zu (%zumb)\n", n, n / mb);
 
-    n = __myst_kernel_args.archive_size;
-    printf("archive size =%11zu (%zumb)\n", n, n / mb);
+    n = __myst_kernel_args.pubkeys_size;
+    printf("pubkeys size =%11zu (%zumb)\n", n, n / mb);
```

Comment:
I think printing the number of public keys / root hashes, rather than the total size, is more meaningful to the user.

Reply:
Perhaps; we should reconsider when we decide whether to enhance the shell feature or take it out.